nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/cleanup/CleanupTaskRIT.java

```diff
@@ -57,12 +58,13 @@
   public static Option[] configureNexus() {
     return NexusPaxExamSupport.options(
         NexusITSupport.configureNexusBase(),
-        nexusFeature("org.sonatype.nexus.plugins", "nexus-repository-r")
+        nexusFeature("org.sonatype.nexus.plugins", "nexus-repository-r"),
+        editConfigurationFileExtend(NEXUS_PROPERTIES_FILE, "nexus.r.packagesBuilder.interval", "1")
```

Comment:
Same as the above

Reply:
done in fe40919080a00ea9162d0af9be02c414af338e27